.woodpecker/.test-pr.yml

```diff
@@ -1,15 +1,15 @@
 steps:
   install:
-    image: danlynn/ember-cli:4.8.0
+    image: danlynn/ember-cli:5.3.0-node_18.18
```

Comment:
We don't really need an ember specific image, just an image with node 18 + chrome but that doesn't seem to exist on Dockerhub yet.